pom.xml

```diff
@@ -1287,6 +1286,12 @@
             <skipLeakTests>false</skipLeakTests>
          </properties>
       </profile>
+      <profile>
+         <id>compatibility-tests</id>
```

Comment:
Would prefer this profile was in the compatibility-tests module. 

Well, it is already there...would prefer the property was redefined in the module.

Reply:
@gemmellr  I thought about doing it the way you mentioned.. but it is how the other properties are defined.. so I kept the way it's currently done. I thought it would be easier to manage the other skips.

Reply:
Those ones at least have the case of applying to lots of modules, and setting the prop is often the only effect of the profile.

This one is entirely specific to compatibility-tests, which already has the profile itself, which does more than just setting the prop, so it would seem nicer if it was all together in the module. (You also just had to make 2 fixup commits that were basically the result of splitting such prop updates).